workers/data_refinery_workers/downloaders/test_array_express.py

```diff
@@ -103,3 +105,53 @@ def test_download_multiple_zips(self, mock_send_job):
         array_express.download_array_express(dlj2.id)
 
         self.assertEqual(ProcessorJob.objects.all().count(), 2)
+
+    @tag('downloaders')
+    def test_dharma(self):
+
+        dlj1 = DownloaderJob()
+        dlj1.accession_code = 'D1'
+        dlj1.worker_id = get_instance_id()
+        dlj1.start_time = datetime.datetime.now()
+        dlj1.save()
+
+        dlj2 = DownloaderJob()
+        dlj2.accession_code = 'D2'
+        dlj2.worker_id = get_instance_id()
+        dlj2.start_time = datetime.datetime.now()
+        dlj2.save()
+
+        dlj3 = DownloaderJob()
+        dlj3.accession_code = 'D3'
+        dlj3.worker_id = get_instance_id()
+        dlj3.save()
+
+        original_file = OriginalFile()
+        original_file.source_url = "ftp://ftp.ebi.ac.uk/pub/databases/microarray/data/experiment/MEXP/E-MEXP-433/E-MEXP-433.raw.1.zip"
+        original_file.source_filename = "Waldhof_020604_R30_01-2753_U133A.CEL"
+        original_file.save()
+
+        assoc = DownloaderJobOriginalFileAssociation()
+        assoc.original_file = original_file
+        assoc.downloader_job = dlj3
+        assoc.save()
+
+        sample = Sample()
+        sample.accession_code = 'Blahblahblah'
+        sample.technology = "MICROARRAY"
+        sample.manufacturer = "AFFYMETRIX"
+        sample.has_raw = True
+        sample.platform_accession_code = "hgu133a"
+        sample.save()
+
+        OriginalFileSampleAssociation.objects.get_or_create(sample=sample, original_file=original_file)
+
+        try:
+            utils.start_job(dlj3.id)
```

Comment:
How does this trigger dharma if there's only 2 tests running before this one is started and the cap is 15?

Reply:
Bad test assertions. Added more robust assertions.